sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,78 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
```

Comment:
No need to predefine variables in python. They are defined the first time you use them.